src/daemon/rpmostreed-os.cxx

```diff
@@ -36,6 +36,8 @@
 #include "rpmostreed-transaction.h"
 #include "rpmostreed-utils.h"
 
+#include <../libdnf/libdnf/hy-util.cpp>
```

Comment:
I think this should just be
`#include <libdnf/libdnf.h>`

Reply:
I get a compiler error when using `include <libdnf/libdnf.h>`: 
```
error: 'hy_is_glob_pattern' was not declared in this scope
``` 
Adding `#include <libdnf/hy-util.cpp>` seems to work though.